mev/mev/settings_production.py

```diff
@@ -20,13 +21,37 @@
 # START logging settings
 ###############################################################################
 
-# setup some logging options for production:
-production_config_dict = copy.deepcopy(log_config.base_logging_config_dict)
-
-# make changes to the default logging dict here:
+# Change the LOGLEVEL env variable if you want logging
+# different than INFO:
+LOGLEVEL = os.environ.get('LOGLEVEL', 'info').upper()
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'console': {
+            'class': 'logging.StreamHandler',
+        },
+    },
+    'root': {
+        'handlers': ['console'],
+        'level':  LOGLEVEL,
+    },
+}
+
+if CLOUD_PLATFORM == AMAZON:
```

Comment:
Would it make sense to factor out AWS-specific settings into a `settings_aws.py` which would import prod settings to avoid adding logic?

Reply:
Don't have strong feelings either way, but we'd then have to alter the provisioning to change `DJANGO_SETTINGS_MODULE` (e.g. in `deployment-aws/puppet/mevapi/data/kvm.yaml`).

There's only a couple spots where we have platform-specific logic, but maybe defer this?